packages/dds/merge-tree/src/test/testClientLogger.ts

```diff
@@ -155,6 +159,12 @@ export class TestClientLogger {
         this.clients.forEach(
             (c) => {
                 if (c === this.clients[0]) { return; }
+                // ensure all clients have seen the same ops
+                assert.equal(
+                    c.getCurrentSeq(),
+                    this.clients[0].getCurrentSeq(),
+                    `Client ${c.longClientId} current seq does not match client ${this.clients[0].longClientId}`,
+                );
```

Comment:
Any value in printing the seq numbers in the message in addition to the client ids?

Reply:
i don't think so. this should only ever fire in test development, and signals that the test code the processes messages is incorrect. the sequence numbers themselves won't tell the developer much without debugging